css/index.css

```diff
@@ -137,10 +175,103 @@ footer nav {
     justify-content: space-between;
     align-items: center;
     padding: 20px 2%;
-    font-size: 14px;
+    font-size: 1.4rem;
 }
 
 footer nav a {
     color: white;
     text-decoration: none;
-}
\ No newline at end of file
+}
+@media (min-width: 1000px){
+	.middle-content .boxes .Box2 {
```

Comment:
Some of your code lines are slightly off in format. A quick fix to this is doing `ctrl-a` and then `alt-shift-f`. If that doesn't work, you can either download an extension from vscode called prettify and or clean it up by hand!